code/modules/mob/mob_defines.dm

```diff
@@ -162,14 +163,8 @@ CREATION_TEST_IGNORE_SELF(/mob)
 	/// Can this mob enter shuttles
 	var/move_on_shuttle = 1
 
-	/**
-	  * construct spells and mime spells.
-	  *
-	  * Spells that do not transfer from one mob to another and can not be lost in mindswap.
-	  * obviously do not live in the mind
-	  */
-	var/list/mob_spell_list = list()
-
+	///A weakref to the last mob/living/carbon to push/drag/grab this mob (exclusively used by slimes friend recognition)
+	var/datum/weakref/LAssailant = null
```

Comment:
This was deleted on purpose, what do you need it for? Just porting a variable declartion won't work when the existing code was removed.

Reply:
sowwy